### PR TITLE
[RM Ch2]: Linewrap to 120 chars

### PR DESCRIPTION
**Pull request by CsatariGergely**
Partially closes #2834

### CODE REVIEW

**Review by ulikleber — APPROVED** [review]

File: doc/ref_model/chapters/chapter02.rst [hunk 1]
```diff
@@ -260,112 +290,220 @@ Telco Edge is commonly coupled with 5G use cases, seen as one of the ingredients
...
++-------------------+-------------------------+---------------+---------------+------------------------+---------------+
+| Profile Extension | Mnemonic                | Applicable to | Applicable to | Description            | Notes         |
+| Name              |                         | Basic Profile | High          |                        |               |
+|                   |                         |               | Performance   |                        |               |
+|                   |                         |               | Profile       |                        |               |
++-------------------+-------------------------+---------------+---------------+------------------------+---------------+
+| Compute Intensive | compute-high-perf-cpu   | ❌            | ✅            | Nodes that have        | May use       |
```
Comment (karinesevilla) [inline doc/ref_model/chapters/chapter02.rst:411]: @CsatariGergely There are extra spaces in some lines of the table, The table is not displayed

**Review by walterkozlowski — APPROVED** [review]

**Comment by gkunz** [comment]
@CsatariGergely can you retrigger the checks?

**Comment by CsatariGergely** [comment]
> @CsatariGergely can you retrigger the checks?

Is there any other way to retrigger, than merge?

**Comment by walterkozlowski** [comment]
@pgoyal01, can you approve please? Thanks.

**Pull request merged**